form_configs/gold_price_prediction.json

```diff
@@ -0,0 +1,32 @@
+{
+  "Gold Price Form": {
+    "SPX": {
+      "type": "number",
+      "min_value": 0.0,
+      "default_value": 1447.160034,
```

Comment:
The values on the forms are rounded off to the last 2 digits, so the updates in the fields aren't noticeable